### ISSUE TITLE
cut neuron issue

### ISSUE THREAD

**Issue by alooow** [schlegelp/navis#9]
Hi, 

I have encountered a problem with the `navis.cut_neuron` function. I know there was recently an update of this function, but still, installing navis directly from this repository (as of 07.10.2020) and using networkx==2.4 gives me te following error:

The code (simply from tutorial):
```
import navis as na
n = na.example_neurons(1).simple
bp = n.nodes[n.nodes.type=='branch'].node_id.values
n_cut = na.cut_neuron(n, bp[0])
```

And the Error:

```
AttributeError                            Traceback (most recent call last)
<ipython-input-7-37dcecfafc6f> in <module>
      2 n = na.example_neurons(1).simple
      3 bp = n.nodes[n.nodes.type=='branch'].node_id.values
----> 4 n_cut = na.cut_neuron(n, bp[0])

~/anaconda3/envs/py3/lib/python3.7/site-packages/navis/graph/graph_utils.py in cut_neuron(x, cut_node, ret)
   1325             cut = _cut_igraph(to_cut, cn, ret)
   1326         else:
-> 1327             cut = _cut_networkx(to_cut, cn, ret)
   1328 
   1329         # If ret != 'both', we will get only a single neuron - therefore

~/anaconda3/envs/py3/lib/python3.7/site-packages/navis/graph/graph_utils.py in _cut_networkx(x, cut_node, ret)
   1426 
   1427         # Reassign graphs
-> 1428         dist.graph = dist_graph
   1429 
   1430         # Clear other temporary attributes

AttributeError: can't set attribute
```

Best,
Alex

**Comment by schlegelp** [comment]
Good catch. Hold on, will fix it right away.

**Comment by schlegelp** [comment]
OK, it should be fixed in the repo now and I'll also release a new version to PyPi asap.

**Comment by alooow** [comment]
Thanks for such a quick fix! It works now.

**Comment by schlegelp** [comment]
No problem, glad you find this package useful! Closed with 0b6a440382215f9e5669c441d846ef1a1f1c5208
